packages/meditrak-app/app/entityMenu/actions.js

```diff
@@ -34,6 +34,20 @@ const getEntityFilters = (state, database, questionId) => {
   return filters;
 };
 
+const getEntityAttributeFilters = (state, questionId) => {
+  const question = getQuestion(state, questionId);
+  const { attributes } = question.config.entity;
+
+  return entity =>
+    attributes
+      ? Object.entries(attributes).every(([key, config]) => {
+          const attributeValue = getAnswerForQuestion(state, config.questionId);
```

Comment:
There's no requirement for how to deal with it, but I'm interested in what happens when there is no answer to the specified question? Do we get all entities or none?

Reply:
That's a good point. In my opinion the right behaviour should be getting all entities as it is equivalent to no filter is applied? I can update it